app/controllers/idv/by_mail/resend_letter_controller.rb

```diff
@@ -83,11 +83,7 @@ def pii_locked?
       end
 
       def step_indicator_steps
-        if in_person_proofing?
```

Comment:
This was probably just missed in the IPP GPO removal, I did some manual testing and this condition should never be true. There is no longer a need for IPP GPO steps so I updated this block.